scripts/init_sms_bridge.py

```
@@ -0,0 +1,117 @@
+    """Initialize database tables"""
+    print("Initializing database tables...")
+    init_db()
+    print("✓ Database tables created")
+
+
+def create_default_settings(created_by: str = "system"):
+    """Create default settings entry"""
+    with get_db_context() as db:
+        # Check if settings exist
+        existing = db.query(SettingsHistory).filter(
+            SettingsHistory.is_active == True
```

Ruff (0.14.10)

Avoid equality comparisons to `True`; use `SettingsHistory.is_active:` for truth checks

Replace with `SettingsHistory.is_active`

(E712)